### ISSUE TITLE
Consider updating PhET girl and man

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/forces-and-motion-basics#137]
We might want to consider updating to our more modern version of PhET girl. Will be a question for the design team, but might be a nice update.

**Comment by samreid** [comment]
Here's the current girl in the sim:

![image](https://cloud.githubusercontent.com/assets/679486/8736260/239ce118-2bd4-11e5-93f9-9cc5cb2e362d.png)

Here's the updated phet girl:

![image](https://cloud.githubusercontent.com/assets/679486/8736277/3624e25e-2bd4-11e5-9ed7-68098baee78d.png)

In my opinion this should be updated if our artist has time.  I think it would be good to update the man character too so they are a similar quality and stylistic level.  Reassigning to @ariel-phet to see if he wants feedback from others or to request the work from the artist.

**Comment by ariel-phet** [comment]
I will request work from the artist.

**Comment by ariel-phet** [comment]
Being worked on in https://github.com/phetsims/tasks/issues/355

**Comment by ariel-phet** [comment]
@samreid this artwork has been updated. Various characters available here:

https://drive.google.com/drive/folders/0B9X7KMQDIh-1alhsWldvT2k1bmM 

in the "sitting phets.ai" file

Please try integrating these into FAMB and let me know if there are any issues.

**Comment by samreid** [comment]
There were graphics for 4 different characters: woman, boy, girl, man.  I used the latter two since they were the kinds already present in the sim:

![image](https://cloud.githubusercontent.com/assets/679486/11433751/f98bfb0c-947e-11e5-94a3-bf840480a14b.png)

This looks much better.  There is one issue that the man's hands don't go up to the top, so there is a gap like this:

![image](https://cloud.githubusercontent.com/assets/679486/11433756/0e495b70-947f-11e5-831e-090b83a47f75.png)

I have made these changes in master and added the AI file (with all 4 characters) to the assets directory.  I published the changes on the dev site here:
http://www.colorado.edu/physics/phet/dev/html/forces-and-motion-basics/1.1.5-dev.12/forces-and-motion-basics_en.html

@ariel-phet can you review this version and let me know if you have other recommendations?  Also, for the issue of the man's hands not going to the top of the image, do you want to request the artist to redraw the hands to go to the top?  Another option would be to write custom code to handle this case, but that could take 30+ minutes and increase the complexity of the code a bit.

**Comment by amyh-phet** [comment]
@ariel-phet  and @samreid   The screen icon for the Motion screen still has the old artwork. Does this need to be updated as well?
![famb_motion_icon](https://cloud.githubusercontent.com/assets/13650180/11569638/31ea3c30-99b0-11e5-877e-157398c70b48.png)

**Comment by ariel-phet** [comment]
I will have Mariah fix up the image

**Comment by ariel-phet** [comment]
Mariah has fixed the images. AI file can be found here:

https://drive.google.com/drive/folders/0B9X7KMQDIh-1alhsWldvT2k1bmM

the file is "sitting phets FIX.ai"

**Comment by samreid** [comment]
Updated graphics above, published here:

http://www.colorado.edu/physics/phet/dev/html/forces-and-motion-basics/1.1.5-dev.13/forces-and-motion-basics_en.html

Reassigned to @ariel-phet to review and to delegate for fixing the homescreen/navbar icons based on @amyh-phet recommendation above.

**Comment by ariel-phet** [comment]
Some slight tweaks need to occur, Mariah will be working on them, tracked still in https://github.com/phetsims/tasks/issues/355

Assigning to @jessegreenberg since this will require replacing the images once the work is complete

**Comment by jessegreenberg** [comment]
New images added, new 'girl-holding' image seems a bit pixelated in sim.  Will try again, double checking export settings.  If that doesn't help, can try saving a larger image scaled with mipmap plugin.

**Comment by jessegreenberg** [comment]
Scaling reduces image quality quite a bit.  Will try mipmaps.

**Comment by jessegreenberg** [comment]
mipmap plugin helps quite a bit.  Here is a dev version with the new images for girl and man added.

http://www.colorado.edu/physics/phet/dev/html/forces-and-motion-basics/1.1.5-dev.14/forces-and-motion-basics_en.html

Assigning to @ariel-phet to review.

**Comment by ariel-phet** [comment]
@jessegreenberg looks good! One tweak...one the motion screen, it looks like the man and girl are not centered on the skateboard (but the crate and other objects appear correct)

![famb](https://cloud.githubusercontent.com/assets/4051174/13190277/7ec35806-d719-11e5-9b94-cacaea4a7457.PNG)

Lets fix that centering issue, and then I think we can close

**Comment by jessegreenberg** [comment]
Looks like a new translation issue, but I am having a hard time reproducing.  Here is what mine looks like:
![translation](https://cloud.githubusercontent.com/assets/6396244/13190853/13cd52a0-d71d-11e5-8255-f6fa80f0e738.PNG)

**Comment by ariel-phet** [comment]
are there working copy changes that did not make it to the dev version?

**Comment by jessegreenberg** [comment]
![capture](https://cloud.githubusercontent.com/assets/6396244/13190966/ba633f4e-d71d-11e5-8d55-f200eee299db.PNG)

After dragging lots of items around, now the girl is off center.  I am going to create a new issue to track this down.  Thanks for catching this!

@ariel-phet before we close this, do you want to handle https://github.com/phetsims/forces-and-motion-basics/issues/137#issuecomment-161738404?

**Comment by ariel-phet** [comment]
@jessegreenberg sounds good

as for https://github.com/phetsims/forces-and-motion-basics/issues/137#issuecomment-161738404 please make a separate issue. We will adjust those once we have new artwork for the pusher from Mariah

**Comment by jessegreenberg** [comment]
Closing this issue since the new images have been added and they look good.  New issues were created for the remaining items above.
